src/hooks/useUserLocation.js

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useState } from 'react';
+import { getDailySummary, groupByDate, sortByDate } from '../utils';
+
+const url = process.env.REACT_APP_API_URL;
+const APPID = process.env.REACT_APP_API_KEY;
+const options = {
+  enableHighAccuracy: true,
+  maximumAge: 0,
+  timeout: 5000,
+};
+
+const useUserLocation = (skip) => {
+  const [position, setPosition] = useState([]);
+  const [data, setData] = useState({ city: null, error: false, loading: true, list: [], summary: [] });
+  const [latitude, longitude] = position;
+
+  useEffect(() => {
+    const handleError = (error) => {
+      const message = typeof error === 'object' ? error.message : error;
+      setData(data => ({ ...data, error: message || true, loading: false }));
+    };
+
+    const handleSuccess = position => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/megwali/punny-weather/pullRequest?prid=7693782)
```suggestion
    const handleSuccess = (position) => {
```